lib/components/artist/shows/index.js

```diff
@@ -0,0 +1,8 @@
+/* @flow */
+'use strict';
+
+import Shows from './shows';
+
+export default {
+  Shows,
+};
```

Comment:
This now doesn’t do anything but jump through a hoop, instead you can remove this one and just rename `shows.js` to `index.js`.